index.html

```diff
@@ -1,5 +1,6 @@
 ---
 layout: default
+prev_page: [youtube.com, EEEE]
```

Comment:
Umm, what's this?

Reply:
the `prev_page` is something snek added in for buttons to the previous page and the next page.
I think the value is there as a placeholder (links to youtube, text on button is EEEE)

Reply:
Shoot I forgot to remove that